dd-java-agent/agent-tooling/src/main/java8/datadog/trace/agent/tooling/bytebuddy/ClassTransformationEvent.java

```diff
@@ -0,0 +1,25 @@
+package datadog.trace.agent.tooling.bytebuddy;
+
+import jdk.jfr.Category;
+import jdk.jfr.Event;
+import jdk.jfr.Label;
+import jdk.jfr.Name;
+
+@Category({"Data Dog Tracer", "Class Transformation"})
```

Comment:
```suggestion
@Category({"Datadog", "Tracer", "Class Transformation"})
```

We already have a bunch of events in `Datadog` category so it would be nice to have this one there as well.

Reply:
Good point! Thanks!